### ISSUE TITLE
Javascript error: Uncaught RangeError

### ISSUE THREAD

**Issue by gmodos** [xws-bench/bench#49]
You can see on attached images. (There isn't fire button. And defence dice.)
![sb_error_21](https://cloud.githubusercontent.com/assets/12628798/25087309/28986a28-236e-11e7-93a1-4b67a7d364a5.png)
![sb_error_22](https://cloud.githubusercontent.com/assets/12628798/25087313/2ede8958-236e-11e7-8df4-95b655e58c79.png)

**Comment by Sleet01** [comment]
This is usually a sign of infinite recursion.  Is there a loop between getresultmodifiers and displayattacktokens2 in xwings.js?

**Comment by xws-bench** [comment]
No, because it's an path executed in all combats. Now this bug must come from an upgrade/pilot. Who was the attacker and defender in this action ? with which upgrades ?
